app/server.go

```diff
@@ -188,6 +188,18 @@ func NewServer(options ...Option) (*Server, error) {
 		s.InitEmailBatching()
 	})
 
+	// Start plugin health check job
+	pluginsEnvironment := s.PluginsEnvironment
+	if pluginsEnvironment != nil {
+		pluginsEnvironment.InitPluginHealthCheckJob(s.Config().PluginSettings.EnableHealthCheck)
+	}
+	s.AddConfigListener(func(_, c *model.Config) {
+		pluginsEnvironment := s.PluginsEnvironment
+		if pluginsEnvironment != nil {
+			pluginsEnvironment.InitPluginHealthCheckJob(c.PluginSettings.EnableHealthCheck)
+		}
+	})
```

Comment:
Will running this in server.NewServer() run this on every instance in a cluster? The EmailBatching job begins this way, which is what I initially modeled the job after.

Reply:
Yes. This should now run on every instance.